test/e2e/grpc/client.test.ts

```diff
@@ -385,4 +386,54 @@ describe('grpc/client', () => {
       await stream2.stop();
     });
   });
-});
+
+  describe('distributed lock', () => {
+    it('should be able to acquire a new lock and unlock', async () => {
+      const tryLock = await client.lock.tryLock("redislock", "resourceId", "owner1", 1000);
+      expect(tryLock.success).toEqual(true);
+      const unlock = await client.lock.unlock("redislock", "resourceId", "owner1");
+      expect(unlock.status).toEqual(LockStatus.Success);
+    });
+
+    it('should be not be able to unlock when the lock is no acquired', async () => {
```

Comment:
Is it possible that this test runs with the test above in parallel? Maybe we can make the resource IDs unique across tests to prevent issues with that.

Reply:
Across all the tests we can actually keep the resource ID unique, maybe creating unique GUIDs?

Reply:
I would like to keep different resource ids. The tests are run in parallel, the following test would fail as the resource occupied by the previous test is not released.